app/helpers/vita_min_form_builder.rb

```diff
@@ -485,4 +517,120 @@ def warning_for_select(element_id, permitted_values, msg)
       'data-permitted': permitted_values.to_json
     )
   end
+
+  # Methods that override honeycriscp v1
+
+  alias v1_cfa_input_field cfa_input_field
+  def cfa_input_field(
+    method,
+    label_text,
+    type: "text",
+    help_text: nil,
+    options: {},
+    autofocus: nil,
+    classes: [],
+    optional: false,
+    prefix: nil
+  )
+    if prefix || options[:'data-mask']
+      return v1_cfa_input_field(
+        method,
+        label_text,
+        type: type,
+        help_text: help_text,
+        options: options,
+        autofocus: autofocus,
+        classes: classes,
+        optional: optional,
+        prefix: prefix,
+        postfix: nil, # not used in this application
+        notice: nil, # not used in this application
+      )
+    end
+
+    text_field_options = options.merge(
+      type: type,
+      autofocus: autofocus,
+      required: options[:required]
+    )
+
+    if !options[:required] && optional
+      text_field_options = text_field_options.merge(required: false)
+    end
+
+    @v2_builder.cfa_text_field(
+      method,
+      label_text,
+      help_text: help_text,
+      wrapper_options: { class: classes.join(' ') },
+      label_options: {},
+      **text_field_options
+    )
+  end
+
+  # Added help text to label and field method instead to remove it from label
+  def label_contents(label_text, help_text, optional: false, include_help_text: true)
+    label_text = <<~HTML
+      <div class="form-question">#{label_text + optional_text(optional)}</div>
+    HTML
+
+    if help_text && include_help_text
+      label_text << <<~HTML
+        <p class="text--help">#{help_text}</p>
+      HTML
+    end
+
+    label_text.html_safe
+  end
+
+  def label_and_field(
```

Comment:
Overrides this honeycrisp method: https://github.com/codeforamerica/honeycrisp-gem/blob/8f20565dd9354362491000ede2bf7c242d7d39a4/app/helpers/cfa/styleguide/cfa_form_builder.rb#L455